### DISCUSSION TITLE
MWS API end date must be extended due to the amount of bugs not being addressed in SP-API

### DISCUSSION THREAD

**Discussion by nrogers805** [amzn/selling-partner-api-models#3392] [General]
Not much more to add - title says it all

**Comment by vaidas-lungis** [comment]
Can you point out couple of them?

**Reply by nrogers805** [reply]
> Can you point out couple of them?

Are you serious!

I am continually raising cases with SP-API support and currently have 5 ! - yes 5 still outstanding

There loads and loads of unanswered issues in the github issues log

The most serious one that I have right now, and has gone on for almost a week is issue #2537 which is severely impacting my business

**Reply by vaidas-lungis** [reply]
Thanks, I just wanted to know what I may face and already know if it's issue with my code / request or something on amz side.
